doc/source/release/1.22.0-notes.rst

```diff
@@ -59,7 +59,7 @@ value for the ``usemask`` parameter.
 Deprecations
 ============
 
-Use delimiter rather than delimitor as kwarg in mrecords
+Use delimiter rather than delimiter as kwarg in mrecords
```

Comment:
I think this still needs to be reverted?

Reply:
https://github.com/numpy/numpy/pull/28569/files

I have reverted all changes to release notes. The change you highlight hear was reverted.

Reply:
Reverted